api/kessel/inventory/v1beta1/policy.proto

```diff
@@ -14,16 +12,10 @@ option java_package = "org.project_kessel.api.inventory.v1beta1";
 
 message Policy {
   // Metadata about this resource
-  Metadata metadata = 1 [ (google.api.field_behavior) = REQUIRED ];
+  Metadata metadata = 1;
 
   // Write only reporter specific data
-  ReporterData reporter_data = 245278792
-      [ (google.api.field_behavior) = INPUT_ONLY ];
-
-  // The entities that registered this item in the Kessel Asset Inventory. The
-  // same resource may be registered by multiple reporters.
-  repeated ReporterData reporters = 353323086
-      [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  ReporterData reporter_data = 245278792;
 
-  PolicyDetail data = 2122698;
+   PolicyDetail data = 2122698;
```

Comment:
nitpick: looks like one space slip in there. Surprised our linter didn't catch it. I guess it doesn't care about protobuf files.

```suggestion
  PolicyDetail data = 2122698;
```